### PR TITLE
Change remote server

### LINKED ISSUE #1: The old server is invalid
The old cloud server we used in the past project is expired. We need to migrate and adapt to new server.

### PR DESCRIPTION
**Pull request by IoNaNO**
Approaching to deadline, I am a little bit nervous now.

### CODE REVIEW

File: build.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "android": {
+        "release": {
+            "keystore": "./my-release-key.keystore",
+            "storePassword": "12345678",
+            "alias": "alias_name",
+            "password" : "12345678",
```
Comment (VenusHui) [inline build.json]: keep password in the secret

File: vite-starter/src/main.ts [hunk 2]
```diff
@@ -18,7 +18,8 @@ import 'vant/lib/index.css'
 // If you want to use ElMessage, import it.
 import "element-plus/theme-chalk/src/message.scss";
 
-axios.defaults.baseURL='https://124.70.41.167:5000';
+export const BASEURL = 'https://150.158.130.111:5000';
+axios.defaults.baseURL='https://150.158.130.111:5000';
```
Comment (VenusHui) [inline vite-starter/src/main.ts]: make sure https protocol support ip access

File: build.json (hunk 1 above)
Comment (IoNaNO) [reply build.json]: fixed in new commit

File: vite-starter/src/main.ts (hunk 2 above)
Comment (IoNaNO) [reply vite-starter/src/main.ts]: this have been tested with backend, it works perfectly.

**Review by VenusHui — APPROVED** [review]

**Pull request merged**